Install-Kite.md

```diff
@@ -10,4 +10,22 @@ curl http://central.maven.org/maven2/org/kitesdk/kite-tools/{{site.version}}/kit
 chmod +x {{site.dataset-command}}
 ```
 
-The first command downloads the Kite tools JAR to a local file named _{{site.dataset-command}}_. The second changes the access rights on the JAR so that the classes in the JAR are executable.
+The first command downloads the Kite tools JAR to a local file named `{{site.dataset-command}}`. The second changes the access rights on the JAR so that the classes in the JAR are executable.
+
+Some versions of CDH come with `{{site.dataset-command}}` installed. However, it might not be the latest version. To check the version, from a terminal window, enter the following command.
+
+```
+{{site.dataset-command}} --version
+```
+
+If the version number doesn't match the version you downloaded, you have two options.
+
+1. Include the path to `{{site.dataset-command}}` when you invoke commands. For example, if the JAR file is in the current directory, use `./{{site.dataset-command}}`.
+
+1. Replace the installed version of `{{site.dataset-command}}`.
+  a. Download `{{site.dataset-command}}` (in this example, it is downloaded to the `~/` directory).
+  b. Replace the old version of `{{site.dataset-command}}` in the existing location.
+    1. For packages, use `sudo cp ~/{{site.dataset-command}} /usr/bin/{{site.dataset-command}}`.
```

Comment:
The file in /usr/bin initializes the java environment and then calls the one in /usr/lib/kite/bin. Replacing the one in the Kite install ensures that package updates work and that the java environment is correct. Otherwise there are more issues.